### PR TITLE
This branch is to initialize project and make components ready to receive data

### PR DESCRIPTION
**Pull request by steveWDamesJr**
## Initialize Project:
-Add React Redux
-Structure application using feature folder with ducks pattern
-Use Reusable components
-Add React Router with two Routes and Links for navigation
-Books default view; shows list of books with Remove button
-Create component Book to display single book and reuse it in component BookList that displays list of books
-Create form for adding book in separate component
-Categories displays Under construction text
-Add some Styles
-Add MIT.md license file

"Debug Linters"

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+const App = () => (
+  <div>
+    <Router>
+      <Navbar />
+      <div className="App">
+        <Routes>
+          <Route path="/" element={<Books />} />
+          <Route path="books" element={<Books />} />
+          <Route path="categories" element={<Categories />} />
```
Comment (TedLivist) [inline src/App.js]: - [x] You have a redundant declaration here. I would recommend you remove the second `Route` line for the **Books** as the first one should be enough.

  You should have something link so:
  ```
  <Route path="/" element={<Books />} />
  <Route path="categories" element={<Categories />} />
  ```

File: src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+const Navbar = () => (
+  <header className="flex justify-between items-center Navbar">
+    <nav className="flex">
+      <ul>
+        <li>
+          <Link className="books" to="/">Bookstore CMS</Link>
+        </li>
+        <li>
+          <Link className="books" to="/books">BOOKS</Link>
```
Comment (TedLivist) [inline src/components/Navbar.js]: - [x] Kindly, change this `Link` to match what we have in the App component like so:
  
  ```
  <Link className="books" to="/">BOOKS</Link>
  ```

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @steveWDamesJr,

Good job so far!
✔️ Good documentation
✔️ App works correctly
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @steveWDamesJr, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**